metabase/metabase
# Fix remaining CSP issues with styles

Related to https://github.com/metabase/metabase-private/issues/73

Fixes remaining issues with `style-src` CSP headers. I've left comments in the PR describing the changes.

How to verify:
- Go through the app, use different visualization, change their settings, use dnd
- Same with admin settings
- Make sure there are no `style-src` errors in the console (you can filter by `style-src`)
